### ISSUE TITLE
Share XDXF dictionaries here

### ISSUE THREAD

**Issue by GD-fix** [soshial/xdxf_makedict#48]
[Thousands of XDXF dictionaries](https://booktracker.org/viewtopic.php?p=211743#211743).

**Comment by soshial** [comment]
Everyone is welcome to share their XDXF dictionaries here.

**Comment by soshial** [comment]
@boydkelly, would you like to share yours?

**Comment by boydkelly** [comment]
Yes here is an xdxf [Jula/French dictionary](https://gitlab.com/ci-dict/dyu-xdxf/-/blob/main/mandenkan/dict.xdxf) under active construction.

The xdxf is transformed into asciidoc with [this xslt](https://gitlab.com/ci-dict/dyu-xdxf/-/blob/main/lexicon-adoc.xsl).
And produces[ this html dictionary](https://coastsystems.net/docs/fr/lexique-dyu/)

The xdxf dictionary also produces the following pages and slides (containing quizes based on the dictionary:

[100 most common words in Ivory Coast Jula.
](https://coastsystems.net/docs/en/kodi-w100/)
[Animals in Ivory Coast Jula.
](https://coastsystems.net/blog/en/2021-04-23-animal/)
[Revealjs quiz](https://coastsystems.net/docs/fr/slides/5words/) (changes every day) also based on the xdxf dictionary

(xslt transforms for above documents are also in the [repo](https://gitlab.com/ci-dict/dyu-xdxf).

**Comment by soshial** [comment]
Here's an [Ithkuil Dictionary](https://github.com/yuorb/IthDict) (ITH-ENG) in XDXF format. Amazing job, @ryanlo713!
